docs/source/manufacturing/3d_printing/troubleshooting/troubleshooting.rst

```diff
@@ -3,7 +3,7 @@ Troubleshooting Common Issues
 
 Here are some hyperlinks to common issues with printers that are worth looking at during a troubleshooting process.
 
-* `Underextrusion and Overextrusion <../specific_skill_guides/temperature_tuning/temperature_tuning.html>`_
-* `Prints Not Sticking <../specific_skill_guides/print_adhesion/print_adhesion.html>`_
-* `Bed Not Level <../specific_skill_guides/bed_leveling/bed_leveling.html>`_
-* `Parts Not Fitting <../designing_and_slicing/tolerancing/tolerancing.html>`_
\ No newline at end of file
+:doc:`Underextrusion and Overextrusion <../specific_skill_guides/temperature_tuning/temperature_tuning>`
```

Comment:
You forgot to keep the bullets, and if you no longer want the bullets you need a space between each line to prevent run-on links.